main.js

```diff
@@ -15,4 +15,4 @@ list.forEach((element) => {
   element.addEventListener('click', closeBtn);
 });
 btn.addEventListener('click', navList);
```

Comment:
- [ ] Make sure the hamburger button and menu follow the design from the chosen template in Figma (e.g., font, colors, images, tests, margins). There is space before the word `About`. Also, the underlines are shorter compared to the original mobile menu. Take a look below screenshots 👇 
- Original design:
![Screenshot (170)](https://user-images.githubusercontent.com/67087664/152156913-29a34087-ddcc-4cbe-a80f-595908b3d944.png)
- Your design design:
![Screenshot (169)](https://user-images.githubusercontent.com/67087664/152156940-cc1d41e8-916f-4112-ab5a-42d1ae5f7454.png)
Kindly fix the differences 🙏